test/integration/targets/nxos_command/tests/cli/sanity.yaml

```diff
@@ -19,7 +19,6 @@
   - assert: &fail
       that:
         - "result.failed == true"
-        - "'Invalid command' in result.msg"
```

Comment:
Do we get any failure reason we can check for?

Reply:
@gundalow resolved.